### PR TITLE
chore: update modal-navigation app to use local modules via symlink

### PR DESCRIPTION
**Pull request by VladimirAmiorkov**
Update the e2e/modal-navigation` application to work with local `tns-core-modules`

### CODE REVIEW

File: e2e/modal-navigation/tsconfig.json [hunk 1]
```diff
@@ -21,7 +21,12 @@
             ]
         }
     },
+    "include": [
+        "../tns-core-modules",
+        "**/*"
+    ],
     "exclude": [
+        "../tns-core-modules/node_modules",
```
Comment (MartoYankov) [inline e2e/modal-navigation/tsconfig.json]: I think the last we agreed is to remove/not use these includes. We are not sure how this works and what it does, but the path is wrong and adding configurations that don't do anything isn't a good idea.

Comment (VladimirAmiorkov) [reply e2e/modal-navigation/tsconfig.json]: I could remove it and.or update it but there is the same here: https://github.com/NativeScript/NativeScript/blob/master/apps/tsconfig.json#L25 . Should we also remove that?

Comment (MartoYankov) [reply e2e/modal-navigation/tsconfig.json]: I believe we discussed that all these should be cleaned up together, but it's not critical at the moment.

Comment (MartoYankov) [reply e2e/modal-navigation/tsconfig.json]: Btw, the app that you are linking to doesn't exist anymore. The apps there have been moved to various places.

Comment (VladimirAmiorkov) [reply e2e/modal-navigation/tsconfig.json]: What do you mean it does not exist anymore? Are you talking about this app: https://github.com/NativeScript/NativeScript/tree/master/e2e/modal-navigation

Comment (VladimirAmiorkov) [reply e2e/modal-navigation/tsconfig.json]: I have removed this above line from the `exclude`

**Pull request closed without merging**